### ISSUE TITLE
int, float, string, and bool must encapsulate bytes

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo#2437]
Let's get rid of `Δ` attribute in `int`, `float`, `string`, and `bool`. Only `bytes` must still have it. The other four objects must encapsulate `bytes`.

Also, it should be possible to make an instance of `int` from the code like this:

```
Q.org.eolang.int
  00-00-00-00-00-00-00-2A
```

This is similar to this:

```
int
  42
```

or just this:

```
42
```

**Comment by maxonfjvipon** [comment]
@yegor256 what about something like this?

1. Rewrite primitives like `int` except `bytes` in such way:
```
[data] > int
  (bytes data).as-int > @!
  ...
```
2. Via `xsl`:
  - `int 00-00-00-00-00-00-00-2A` => `new PhWith(new EOint(Phi.Φ), 0, new Data.Value<>(new byte[] {...}))`
  - `int 42` => `new PhWith(new EOint(Phi.Φ), 0, new Data.Value<>(new BytesOf(42L).take())`
  - `42` => `new PhWith(new EOint(Phi.Φ), 0, new Data.Value<>(new BytesOf(42L).take())`

**Comment by 0pdd** [comment]
@yegor256 8 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2589](https://github.com/objectionary/eo/issues/2589), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591) are still not solved.

**Comment by maxonfjvipon** [comment]
@yegor256 let's make a new release with tag `0.33.0`

**Comment by 0pdd** [comment]
@yegor256 7 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2588](https://github.com/objectionary/eo/issues/2588), [#2589](https://github.com/objectionary/eo/issues/2589), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591) are still not solved; solved: [#2587](https://github.com/objectionary/eo/issues/2587).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2586](https://github.com/objectionary/eo/issues/2586), [#2589](https://github.com/objectionary/eo/issues/2589), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588).

**Comment by 0pdd** [comment]
@yegor256 4 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2589](https://github.com/objectionary/eo/issues/2589), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588).

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2589](https://github.com/objectionary/eo/issues/2589), [#2591](https://github.com/objectionary/eo/issues/2591) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2590](https://github.com/objectionary/eo/issues/2590).

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2589](https://github.com/objectionary/eo/issues/2589), [#2645](https://github.com/objectionary/eo/issues/2645) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591).

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2589](https://github.com/objectionary/eo/issues/2589) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591), [#2645](https://github.com/objectionary/eo/issues/2645).

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2589](https://github.com/objectionary/eo/issues/2589), [#2819](https://github.com/objectionary/eo/issues/2819) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591), [#2645](https://github.com/objectionary/eo/issues/2645).

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#2584](https://github.com/objectionary/eo/issues/2584), [#2819](https://github.com/objectionary/eo/issues/2819), [#2837](https://github.com/objectionary/eo/issues/2837) are still not solved; solved: [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2589](https://github.com/objectionary/eo/issues/2589), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591), [#2645](https://github.com/objectionary/eo/issues/2645).

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#2819](https://github.com/objectionary/eo/issues/2819), [#2837](https://github.com/objectionary/eo/issues/2837) are still not solved; solved: [#2584](https://github.com/objectionary/eo/issues/2584), [#2585](https://github.com/objectionary/eo/issues/2585), [#2586](https://github.com/objectionary/eo/issues/2586), [#2587](https://github.com/objectionary/eo/issues/2587), [#2588](https://github.com/objectionary/eo/issues/2588), [#2589](https://github.com/objectionary/eo/issues/2589), [#2590](https://github.com/objectionary/eo/issues/2590), [#2591](https://github.com/objectionary/eo/issues/2591), [#2645](https://github.com/objectionary/eo/issues/2645).
